src/bicep/examples/zeroTrustWorkbook/README.md

```diff
@@ -0,0 +1,81 @@
+
+The following table lists the required parameters for a Mission LZ deployment to enable an Azure Sentinel Workbook:
+
+Required Parameters | Description
+------------------- | -----------
+_location_ | The region to deploy Azure Sentinel into
+_resourcePrefix_ | A 3-10 alphanumeric character string without whitespace, used to prefix resources and generate uniqueness for resources with globally unique naming requirements
+_deploySentinel_ | A boolean expression indicating that Azure Sentinel is to be deployed with the MissionLZ deployment
+_deployDefender_ | A boolean expression indicating that Microsoft Defender for Cloud is enabled in the Mission LZ deployment
+
+An example deployment with required deployment parameters included is shown below:
+
+```bash
+
+ az deployment sub create \
+   --subscription $deploymentSubscription \
+   --name "myMlzDeployment" \
+   --template-file ./mlz.bicep \
+   --parameters resourcePrefix=myPrefix \
+   deploySentinel=true \
+   deployDefender=true
+```
+
+## Deploying Sentinel Zero Trust (TIC3.0) Workbook
+
+The Sentinel Zero Trust (TIC3.0) Workbook is maintained in the [Azure Sentinel GitHub repository](https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ZeroTrust(TIC3.0)/readme.md)
+
+With the link provided, it is possible to use the "Deploy to Azure" button with some simple input parameters for Azure Government and Azure Commercial clouds.
+
+From the pre-existing deployment of MLZ shown above, the following parameters are required for deployment of the Azure Sentinel Workbook:
+
+ Required Input Parameters | Description
+---------------------------|------------
+_operationsSubscriptionId_ | The subscription that contains the Log Analytics Workspace and to deploy the Sentinel solution into
+_operationsResourceGroupName_ | The resource group that contains the Log Analytics Workspace to link Azure Sentinel to
+_logAnalyticsWorkspaceName_ | The name of the Log Analytics Workspace to link Azure Sentinel to
+
+The `$operationsResourceGroupName` utilizes the `$resourcePrefix` in a typical Mission LZ deployment. In the example deployment above, the $resourcePrefix is `myPrefix`. The standard naming convention of the example Operations resource group will be:
+
+`myPrefix-rg-operations-mlz`
+
+This can be searchable through the Azure CLI:
+
+```bash
+az group list --query [].name --out tsv | grep "operations"
+```
+
+To retrieve the `$logAnalyticsWorkspaceName`, the following naming convention will be adhered to in a typical Mission LZ deployment with the example provided:
+
+`myPrefix-log-operations-mlz`
+
+This parameter is also searchable with the Azure CLI:
+
+```bash
+az monitor log-analytics workspace list --query [].name --out tsv --resource-group $operationsResourceGroupName
+```
+
+To deploy the workbook through Azure CLI (with the example parameters provided for the required variables):
```

Comment:
The table lists input parameters, but they are not parameters to deploy the workbook, which is confusing. I suggest removing the snippets for finding the workspace name and replacing it with a simple description. An example is below, but please apply additional edits as you see fit.
```suggestion

### Command Line Workbook Deployment

The workbook can be deployed using the Azure CLI `az deployment` command. The workbook template requires the `workspace` parameter, which is the name of the Log Analytics workspace connected to Sentinel in MLZ. The workspace name can be found in the MLZ operations resource group, which also contains the Log Analytics and Sentinel deployment. The same resource group is where the `az deployment` command is scoped. See the example below:
```